README.md

```diff
@@ -21,18 +35,28 @@ This repository is a fork of [ChromeDevTools/devtools-frontend](https://github.c
 
 1. Build continuously with a file watcher:
 
+   This option continuously rebuilds the project whenever a file changes. It's best suited for active development as it allows you to see changes
+   in real time, but it can consume more system resources, especially in larger projects. Use this during iterative work when you need fast
+   feedback.
+
     ```sh
     npm run watch
     ```
 
 1. Build with the default config once:
 
+   This single, non-watched build compiles the project with the default configuration. It’s quicker and less resource-heavy than the watch build,
+   making it ideal for testing changes without long-term commitment to resource usage.
+
     ```sh
     npm run build
     ```
 
 1. Build with the release config once:
 
+   This one-time build optimizes the code for production use. It’s slower to compile and can take significantly more time but results in a stable,
+   high-performance version suitable for production environments. Use this when preparing the project for final deployment or release.
+
```

Comment:
These bits feel too verbose (AI generated?) — these scripts are self-documenting. Intend to come back and merge this PR, but let's do a pass of the file which removes these.